cmd/controller-manager/app/controller-manager.go

```diff
@@ -67,9 +69,18 @@ member clusters and does the necessary reconciliation`,
 		},
 	}
 
+	// Add the command line flags from other dependencies, but do not
```

Comment:
What dependencies, and why is normalization required?

Reply:
I removed those lines and things still worked, so I think they should be removed.   YAGNI (you ain't gonna need it) is a general principle of software development to avoid doing work until it is actually needed.

Reply:
done:)